triangle/triangle-test.js

```diff
@@ -0,0 +1,15 @@
+const assert = require('chai').assert
+const isTriangle = require('./triangle')
+
+
+describe('isTriangle functionality', function() {
+  context('isTriangle function', function(){
+    it('it can return false or true when length of triangle changes', function(){
```

Comment:
If you have the word "or" in your test description, that's a good sign to make two tests out of it. In this case, one    for valid triangles and one for invalid values.